Tests/Integration/inc/functions/rocketCleanCacheDir.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace WP_Rocket\Tests\Integration\inc\functions;
+
+use WP_Rocket\Tests\Integration\FilesystemTestCase;
+
+/**
+ * @covers ::rocket_clean_cache_dir
+ * @group Functions
```

Comment:
Add the Rocket functions that are used in this function like this:

```suggestion
 * @uses :: rocket_direct_filesystem
 * @uses :: rocket_rrmdir
 * @uses :: _rocket_get_cache_path_iterator
 * @uses :: _rocket_get_wp_rocket_cache_path 
 *
 * @group Functions
```

Why? Code coverage static analyzers use these.